fibonacci/main.go

```diff
@@ -3,12 +3,29 @@ package main
 import "fmt"
 
 func main() {
-	fmt.Println(fibonacci(7))
+	const n = 5
+	fmt.Println("Calculating %d sequence\n", n)
```

Comment:
Don't change the contents of the `main` function, fill in only the contents of `fibonacci`.

The `fmt.Println` function does not use substitution codes like `%d` - for that, you need `fmt.Printf`. What `fmt.Println` does is to print each of its arguments, literally, followed by a newline (which in `fmt.Printf` you need to supply as `\n`). The `f` in `Printf` stands for "formatted."

One of Go's biggest strengths is how well its standard library is documented. See [fmt.Printf](https://pkg.go.dev/fmt#Printf) and [fmt.Println](https://pkg.go.dev/fmt#Println).